### PR TITLE
feat(website): emote page

### PR DESCRIPTION
**Pull request by ZonianMidian**
## Proposed changes

Implement emote page.
## Types of changes

What types of changes does your code introduce to 7TV?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/SevenTV/SevenTV/blob/main/CONTRIBUTING.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

[WEB-10](https://sharing.clickup.com/4793979/v/7-4793979-2/t/h/c/4793979/WEB-10/84dc41ba7706e0f)

https://github.com/SevenTV/SevenTV/assets/57513632/270f1672-0aa2-4ac1-ae79-1d2d11acd177

### CODE REVIEW

**Comment by TroyKomodo** [comment]
Task linked: [WEB-10 Emote page](https://app.clickup.com/t/4793979/WEB-10)

File: website/src/routes/emote/[id]/+layout.svelte [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+							<span class="emote-name">
+								{data.name}
+							</span>
+						</div>
+						<span class="time">
+							1 hour ago
+						</span>
+					</div>
+				</div>
+				<hr>
```
[the remark this answers is not in the document]
Comment (ZonianMidian) [reply website/src/routes/emote/[id]/+layout.svelte:162]: > Can this be converted into a component and rendered based on mock data returned via the mock function or some array?

Of course
image

**Review by lennartkloock — APPROVED** [review]
Looks good! Thanks for the help! I changed some styling things but nothing big

File: website/src/routes/emote/[id]/+layout.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (ZonianMidian) [reply website/src/routes/emote/[id]/+layout.svelte]: Implemented in: [91fb315](https://github.com/SevenTV/SevenTV/pull/15/commits/91fb3153ea974a119cec3bf441920564a68d8918)

**Pull request merged**